homeassistant/components/advantage_air/light.py

```diff
@@ -0,0 +1,91 @@
+"""Light platform for Advantage Air integration."""
+from typing import Any
+
+from homeassistant.components.light import ATTR_BRIGHTNESS, ColorMode, LightEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity import DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import (
+    ADVANTAGE_AIR_STATE_OFF,
+    ADVANTAGE_AIR_STATE_ON,
+    DOMAIN as ADVANTAGE_AIR_DOMAIN,
+)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config_entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up AdvantageAir light platform."""
+
+    instance = hass.data[ADVANTAGE_AIR_DOMAIN][config_entry.entry_id]
+
+    entities = []
+    if "myLights" in instance["coordinator"].data:
+        for light in instance["coordinator"].data["myLights"]["lights"].values():
+            if light.get("relay"):
+                entities.append(AdvantageAirLight(instance, light))
+            else:
+                entities.append(AdvantageAirLightDimmable(instance, light))
+    async_add_entities(entities)
+
+
+class AdvantageAirLight(CoordinatorEntity, LightEntity):
+    """Representation of Advantage Air Light."""
+
+    _attr_has_entity_name = True
+    _attr_supported_color_modes = {ColorMode.ONOFF}
+
+    def __init__(self, instance, light):
+        """Initialize an Advantage Air Light."""
+        super().__init__(instance["coordinator"])
+        self.async_set_light = instance["async_set_light"]
+        self._id = light["id"]
+        self._attr_unique_id = f'{self.coordinator.data["system"]["rid"]}-{self._id}'
+        self._attr_device_info = DeviceInfo(
+            identifiers={(ADVANTAGE_AIR_DOMAIN, self._attr_unique_id)},
+            via_device=(ADVANTAGE_AIR_DOMAIN, self.coordinator.data["system"]["rid"]),
+            manufacturer="Advantage Air",
+            model=light.get("moduleType", "Unknown"),
```

Comment:
```suggestion
            model=light.get("moduleType"),
```

Why not just return `None` when the model is not known? HA can handle this and will also localize this, where 'Unknown' cannot be localized.

Reply:
Good catch, this has been implemented as recommended.